tests/v2/validation/charts/istio_test.go

```diff
@@ -0,0 +1,293 @@
+package charts
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/rancher/norman/types"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/charts"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/namespaces"
+	"github.com/rancher/rancher/tests/framework/extensions/workloads/deployments"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/rancher/rancher/tests/integration/pkg/defaults"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type IstioTestSuite struct {
+	suite.Suite
+	client              *rancher.Client
+	session             *session.Session
+	project             *management.Project
+	chartInstallOptions *chartInstallOptions
+}
+
+func (i *IstioTestSuite) TearDownSuite() {
+	i.session.Cleanup()
+}
+
+func (i *IstioTestSuite) SetupSuite() {
+	testSession := session.NewSession(i.T())
+	i.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(i.T(), err)
+
+	i.client = client
+
+	//Get clusterName from config yaml
+	clusterName := client.RancherConfig.ClusterName
+	require.NotEmptyf(i.T(), clusterName, "Cluster name to install is not set")
+
+	//Get clusterID with clusterName
+	clusterID, err := clusters.GetClusterIDByName(client, clusterName)
+	require.NoError(i.T(), err)
+
+	//Change kiali and jaeger paths if it's not local cluster
+	if clusterID != clusterName {
+		kialiPath = fmt.Sprintf("k8s/clusters/%s/%s", clusterID, kialiPath)
+		tracingPath = fmt.Sprintf("k8s/clusters/%s/%s", clusterID, tracingPath)
```

Comment:
@bmdepesa will ever test this on local? If not then can remove this if statement.

Reply:
I think we can also install monitoring in the `local` cluster but `local` is only accessible by Admin.

Reply:
I guess what I'm asking is if we are expected to test against the local cluster